src/utilities/inputProcessors.ts

```diff
@@ -0,0 +1,80 @@
+import { getInput } from '@actions/core';
+import * as joi from '@hapi/joi';
+import { BranchSpec } from 'semantic-release';
+
+enum InputParameters {
+  DryRun = 'dry-run',
+  ReleaseBranches = 'release-branches',
+  CommitAssets = 'commit-assets',
+  ReleaseAssets = 'release-assets',
+  NodeModule = 'node-module',
+}
+
+const inputReleaseBranchesSchema = joi
+  .array()
+  .items(
+    joi.string(),
+    joi.object().keys({
+      channel: joi.alternatives().try(joi.string(), false).optional(),
+      name: joi.string().min(1).required(),
+      prerelease: joi.string().optional(),
```

Comment:
According to https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#prerelease this can also be a boolean.

Reply:
This might be a case of documentation falling behind?

Types are clear on the matter: 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f5b16826d94fc235f7c9e6e557f547c63a134f49/types/semantic-release/index.d.ts#L287

Reply:
Types are not always accurate. When in doubt always go to the library itself...
https://github.com/semantic-release/semantic-release/blob/0785a844fa8ac1320383452ce531898be3b01f92/lib/branches/normalize.js#L94

https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/pre-releases.md

Reply:
It doesn't really mean that we _have to_ support it. We tend to have rather strict code and configuration, so I'd be in favor of dropping these aliases 🤷‍♂️